runtime/queries/column_desc_stats.go

```diff
@@ -55,7 +55,7 @@ func (q *ColumnDescriptiveStatistics) Resolve(ctx context.Context, rt *runtime.R
 	}
 	defer release()
 
-	sanitizedColumnName := safeName(olap.Dialect(), q.ColumnName)
+	sanitizedColumnName := safeName(q.ColumnName)
```

Comment:
```suggestion
	sanitizedColumnName := olap.Dialect().EscapeIdentifier(q.ColumnName)
```

safeName internally uses `EscapeIdentifier` so it is fine to use that directly.